Lombiq.HelpfulLibraries.SourceGenerators/Lombiq.HelpfulLibraries.SourceGenerators/Readme.md

```diff
@@ -0,0 +1,60 @@
+# Lombiq.HelpfulLibraries.SourceGenerators
+
+A collection of helpful source generators.
+**When using one of the generators you must run a build before errors will go away**
```

Comment:
We typically use this textbook style format for warnings. Also you missed the period at the end.

```suggestion

> ⚠ When using one of the generators you must run a build before errors will go away.
```